book/src/guide/run.md

```diff
@@ -34,6 +47,24 @@ To do so add the following to `Cargo.toml`:
 enabled = true
 ```
 
+### Refinement Annotations on a Package
```

Comment:
I was thinking that package is the wrong name for this. Maybe we should call it a "Cargo Project"